toolkit/scripts/toolchain/container/Dockerfile

```diff
@@ -56,14 +56,14 @@ COPY [ "./toolchain-sha256sums", \
 # the naming convention of the source cache (convention used to prevent naming collisions).
 WORKDIR $LFS/sources
 RUN wget -nv --no-clobber --timeout=30 --no-check-certificate --continue --input-file=$LFS/tools/toolchain-local-wget-list --directory-prefix=$LFS/sources; exit 0
-RUN wget -nv --no-clobber --timeout=30 --continue --input-file=$LFS/tools/toolchain-remote-wget-list --directory-prefix=$LFS/sources; exit 0
-RUN wget -nv --no-clobber --timeout=30 --continue https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/mariner/5.15.18.1.tar.gz -O kernel-5.15.18.1.tar.gz --directory-prefix=$LFS/sources; exit 0
+RUN wget -nv --no-clobber --timeout=30 --no-check-certificate --continue --input-file=$LFS/tools/toolchain-remote-wget-list --directory-prefix=$LFS/sources; exit 0
```

Comment:
Would it be possible to install `ca-certificates` into the container instead of skipping the cert check here?

Reply:
I did install that package above but still saw "cannot verify certificate" errors.

Reply:
However, after trying again locally, the verify errors do seem to go away with ca-certificates installed. Let me take a close look at this.

Reply:
Turns out that adding the ca-certificates does allow all of the downloads. I've removed the --no-check-certificate flag. Thanks for pointing this out.

Reply:
Neat, thanks for doing that!